common/server_room.cpp

```diff
@@ -36,6 +36,33 @@ Server_Room::~Server_Room()
     usersLock.unlock();
 }
 
+bool Server_Room::userMayJoin(const ServerInfo_User & userInfo)
+{
+    
+    ServerInfo_Room roomInfo;
```

Comment:
Is there a reason this is here if we never use it? Or do you use it implicitly and I didn't notice?